src/components/Button/Button.tsx

```diff
@@ -62,13 +62,14 @@ export class Button extends React.Component<IButtonProps, IButtonState> {
 
     return React.createElement(
       tag,
-      assign({
-        'aria-label': ariaLabel,
-        'aria-labelledby': ariaLabel ? null : labelId,
-        'aria-describedby': ariaDescription ? ariaDescriptionId : description ? descriptionId : null,
-        'ref': (c: HTMLButtonElement): HTMLButtonElement => this._buttonElement = c
-      },
-        this.props,
+      assign(
+        this.props.rootProps,
```

Comment:
I think you should add a {} to the beginning here, otherwise you're mixing things into the rootProps that the user passes in, which could be empty.